mne/source_estimate.py

```diff
@@ -910,6 +909,24 @@ def transform(self, func, idx=None, tmin=None, tmax=None, copy=False):
         return stcs
 
 
+def _center_of_mass(vertices, values, hemi, subject, subjects_dir,
+                    restrict_vertices):
+    """Helper to find the center of mass on a surface"""
+    subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
+    surf = read_surface(op.join(subjects_dir, subject, 'surf',
+                                hemi + '.sphere'))
+    if restrict_vertices is True:
+        restrict_vertices = vertices
+    elif restrict_vertices is False:
+        restrict_vertices = np.arange(surf[0].shape[0])
```

Comment:
...but maybe we should get rid of `ndarray` because 1) originally I had the same reaction as you, 2) I *highly* doubt anyone is actually using it, and 3) the same thing can be achieved by modifying the label itself properly (although it is a bit more annoying)